Packs/Zscaler/Integrations/Zscaler/Zscaler.py

```diff
@@ -416,123 +419,116 @@ def url_lookup(args):
     multiple = args.get('multiple', 'true').lower() == 'true'
     response = lookup_request(url, multiple)
     raw_res = json.loads(response.content)
-    ec = dict()  # type: Dict[str, List]
-    ec[outputPaths['url']] = []
-    ec['DBotScore'] = []
-    pre_table_data = []
+
     urls_list = argToList(url)
+    results: List[CommandResults] = []
+
     for data in raw_res:
-        suspicious_categories = ['SUSPICIOUS_DESTINATION', 'SPYWARE_OR_ADWARE']
         res_url = data.get('url')
         for url in urls_list:
             # since zscaler expects to recieve a URL without the protocol, we omit it in `lookup_request`
             # in the response, the URL is returned as it was sent, so we add back the protocol by replacing
             # the URL retruned with the one we got as an argument
             if 'http://' + res_url in url or 'https://' + res_url in url:
                 data['url'] = url
+
         ioc_context = {'Address': data['url'], 'Data': data['url']}
         score = 1
```

Comment:
Can all the score values (0, 1, 2, 3) be in an ENUM? And be given meaningful names?